src/test/java/com/example/project/test/ShoppingListTest.java

```diff
@@ -0,0 +1,140 @@
+package com.example.project.test;
+
+import com.example.project.item.Item;
+import com.example.project.page_object.AllListsPage;
+import com.example.project.page_object.ListPage;
+import io.appium.java_client.android.AndroidDriver;
+import io.appium.java_client.android.AndroidElement;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.TimeoutException;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.openqa.selenium.support.ui.FluentWait;
+import org.openqa.selenium.support.ui.Wait;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.time.Duration;
+import java.util.concurrent.TimeUnit;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+public class ShoppingListTest {
+
+    AndroidDriver<AndroidElement> driver;
+    Wait wait;
+
+    @BeforeClass
+    public void setUp() throws MalformedURLException {
```

Comment:
I would suggest to create a BaseTest class and move common logic (setup/tear down) to it. There is no benefit for a single test class, but if you have more that might be a good thing to consider.